starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,43 @@
+body{
+  font-family: 'Helvetica';
+  
+}
+
+h1{
+  color: white;
+  background-image: url('../images/apple-pie.jpg');
+  height: 500px;
+  width: 80%;
+  background-size: cover;
+  text-align: center;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  line-height: 450px;
+  font-size: 75px;
+}
+
+img {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+hr{
+  width: 75%;
+}
+
```

Comment:
too many repeating styles. A good work around is to wrap the tags that are together in a div and then adjust the margin and/or padding of the div